packages/server/src/unstable-core-do-not-import/middleware.ts

```
@@ -1,7 +1,7 @@
 import { TRPCError } from './error/TRPCError';
 import type { ParseFn } from './parser';
 import type { ProcedureType } from './procedure';
-import type { GetRawInputFn, Overwrite, Simplify } from './types';
+import type { GetRawInputFn, Overwrite, Simplify, TypeError } from './types';
```

Biome (1.9.4)

[error] Do not shadow the global "TypeError" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)